test_runner/regress/test_timeline_detach_ancestor.py

```diff
@@ -606,6 +608,41 @@ def test_timeline_ancestor_detach_errors(neon_env_builder: NeonEnvBuilder, shard
     assert e.value.status_code == 404
 
 
+@pytest.mark.parametrize("sharded", [True, False])
+def test_timeline_ancestor_detach_errors(neon_env_builder: NeonEnvBuilder, sharded: bool):
```

Comment:
A test with a short client timeout followed by a retry would be interesting.

Reply:
Ah, I think you are proposing this in the base pr:
```
# TEST: 1. tad is started, one node stalls, other restarts
#       2. client timeout before stall over
#       3. on retry with stalled and other being able to proceed
```

Reply:
The `test_sharded_timeline_detach_ancestor` now includes launching initial request, which gets stuck, then retrying on 503 until it succeeds. Already in review.